docs/en/academic/custom-templates.md

```diff
@@ -1,83 +1,134 @@
+
+If Pandoc exited with an error, Zettlr will show you the error message, which you can cut-and-paste into Google to troubleshoot the problem. If the error occured in LaTeX, the full LaTeX console output will be displayed. This can be quite verbose and confusing for newcomers. For example, if a LaTeX package is missing, you need to spot the relevant error message (i.e. `File <package-name>.sty is missing`) amongst all the other output text.
+
+> **Attention:**  If you use the default template and avoid using LaTeX commands in your Markdown file, you should not encounter any errors. If you do, it indicates there is a problem with the default template, which should be reported to Zettlr's maintainers. 
 
 ## Getting Started with Templating
 
-Now it's time to get to building a template! You can either use an external editor to write your template or use Zettlr itself. Zettlr allows you to write LaTeX templates directly inside it, so you don't have to use an external editor for that.
+Let's write your first LaTeX template, which can be done directly in Zettlr! When complete, your template file will be passed to Zettlr, Citeproc (if applicable), Pandoc, and then finally LaTeX. 
+
+<!-- TO CHECK: On 2.0.0.0-beta on a Linux machine, I do not see a + symbol to add a new file where I can type in the filename with extension. Also, if I select   File -> New File , A file is automatically generated with an ID as its name; it is not intuitive that one has to add some text, then try to save the file in order to be able to rename the file to, for example, 'my-template.tex'. Is this the intended way to create a new file with a specific name?
 
-> Most of you might want to simply take the default template and begin customizing it instead of writing your own from scratch. [The default Pandoc template can be found here](https://github.com/jgm/pandoc/blob/master/data/templates/default.latex).
+A new figure is likely needed if the + menu item is in fact gone; I am happy to make this change if my current interface, without the + is in fact correct.-->
 
 ![Create TeX-Files by appending the appropriate extension](../img/create_tex_file.png)
 
-If you create a new file, but **provide `.tex` as the file extension**, Zettlr will not create a Markdown file but an actual LaTeX file! This file will be indicated with a small `TeX` indicator in the file list (if file meta is turned on) and can be edited from within Zettlr. Hooray!
+First, create a new file and **give it a `.tex` file extension**.  A small `TeX` indicator will appear beneath the filename in the file list.
 
+<!-- TO CHECK: If I save a file with a file extension of .tex, then Zettlr will show the small 'TeX' indicator. However, if you rename an existing file and give it a .tex file extension, Zettlr will now show a small 'TeX' indicator. I would have expected the 'TeX' indicator to be show in both circumstances. 
+
```

Comment:
The "plus" symbol is gone, files are now created much more in line with common text editors (i.e. only using the shortcut or the menu item, not with the toolbar)

Reply:
Oh, and please feel free to just replace figures! Most of them need to be replaced either way!